test/end-to-end/fee_center/feeCenter.page.js

```diff
@@ -0,0 +1,77 @@
+/* jshint expr:true */
```

Comment:
I followed the page object as detailed from the official protractor tutorial, and now my buttons, radio button, and text field are privates members of my page. this let me represent a structure of my page by reading the code I can guess how many element I have on the page and I many actions or methods (interface) the page gives me.